resources/js/Pages/Home.vue

```diff
@@ -3,6 +3,14 @@
     <Menu />
     <Head title="Home" />
     <section class="hero is-dark is-fullheight-with-navbar">
+        <b-message
```

Comment:
Just to have a better hierarchy, I would move the `b-message` out of the section and put it between Head and section (Line 4-5)